app/src/main/java/chat/rocket/android/members/ui/MembersFragment.kt

```diff
@@ -27,32 +27,46 @@ import kotlinx.android.synthetic.main.app_bar_chat_room.*
 import kotlinx.android.synthetic.main.fragment_members.*
 import javax.inject.Inject
 
-fun newInstance(chatRoomId: String): Fragment = MembersFragment().apply {
-    arguments = Bundle(1).apply {
-        putString(BUNDLE_CHAT_ROOM_ID, chatRoomId)
+
+fun newInstance(chatRoomId: String, isOwner: Boolean, isMod: Boolean): Fragment {
```

Comment:
`isModerator` sounds better than `isMod`. You have even created a function with that name on `ChatRoomPresenter.kt` for instance.